ingestfile

```diff
@@ -26,26 +12,22 @@ _usage(){
     echo "This script will run an interactive interview and then process an input file accordingly."
     echo "Dependencies: ${DEPENDENCIES[@]}"
     echo "Usage: $(basename ${0}) [ -options ]"
-    echo "   default uses a graphical user interface for interactive component of inputing file and media ID, setting volume adjustment, formula, and slate creation."
-    echo "  -e Uses the terminal interface for interactive component of inputing file and Media ID, volume adjustment, formula, and slate creation."
```

Comment:
If `-e` is removed then the README should be updated (probably regarding some other options too).